### PR TITLE
Consolidate the rest of se23 jsx logic

### PR DESCRIPTION
**Pull request by sophschneider**
### WHAT is this pull request doing?

* Consolidates `se23` logic in tsx files that were either missed or added back in as part of bad rebases

### How to 🎩

Tophatting links are in the file changes per component

### CODE REVIEW

**Comment by sophschneider** [comment]
UI baseline changes are unrelated to this PR, they match next. I've accepted them here and on next

File: polaris-react/src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.tsx [hunk 1]
```diff
@@ -12,7 +12,6 @@ import type {ContextualSaveBarProps} from '../../../../utilities/frame';
 import {getWidth} from '../../../../utilities/get-width';
```
Comment (sophschneider) [inline polaris-react/src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.tsx:12]: [This PR](https://5d559397bae39100201eedc1-tnpnnmnlft.chromatic.com/?path=/story/all-components-contextualsavebar--default)
[Production](https://storybook.polaris.shopify.com/?path=/story/all-components-contextualsavebar--default&globals=polarisSummerEditions2023:true)

File: polaris-react/src/components/Frame/components/Toast/Toast.tsx [hunk 2]
```diff
@@ -1,9 +1,5 @@
 import React, {useEffect} from 'react';
```
Comment (sophschneider) [inline polaris-react/src/components/Frame/components/Toast/Toast.tsx:1]: To see toast, make a change and save

[This PR](https://5d559397bae39100201eedc1-tnpnnmnlft.chromatic.com/?path=/story/all-components-frame--in-an-application)
[Production](https://storybook.polaris.shopify.com/?path=/story/all-components-frame--in-an-application&globals=polarisSummerEditions2023:true)

File: polaris-react/src/components/FullscreenBar/FullscreenBar.tsx [hunk 3]
```diff
@@ -4,7 +4,6 @@ import {ExitMajor} from '@shopify/polaris-icons';
 import {Icon} from '../Icon';
```
Comment (sophschneider) [inline polaris-react/src/components/FullscreenBar/FullscreenBar.tsx:4]: [This PR](https://5d559397bae39100201eedc1-tnpnnmnlft.chromatic.com/?path=/story/all-components-fullscreenbar--all)
[Production](https://storybook.polaris.shopify.com/?path=/story/all-components-fullscreenbar--all&globals=polarisSummerEditions2023:true)

File: polaris-react/src/components/ResourceItem/ResourceItem.tsx [hunk 4]
```diff
@@ -19,7 +19,6 @@ import {useBreakpoints} from '../../utilities/breakpoints';
 import type {BreakpointsDirectionAlias} from '../../utilities/breakpoints';
```
Comment (sophschneider) [inline polaris-react/src/components/ResourceItem/ResourceItem.tsx:19]: [This PR](https://5d559397bae39100201eedc1-tnpnnmnlft.chromatic.com/?path=/story/all-components-resourceitem--default)
[Production](https://storybook.polaris.shopify.com/?path=/story/all-components-resourceitem--default&globals=polarisSummerEditions2023:true)

File: polaris-react/src/components/Tabs/Tabs.tsx [hunk 5]
```diff
@@ -480,10 +474,7 @@ export const Tabs = ({
             styles['IconWrap-open'],
         )}
       >
-        <Icon
-          source={polarisSummerEditions2023 ? ChevronDownMinor : CaretDownMinor}
-          tone="subdued"
-        />
+        <Icon source={ChevronDownMinor} tone="subdued" />
```
Comment (sophschneider) [inline polaris-react/src/components/Tabs/Tabs.tsx:477]: [This PR](https://5d559397bae39100201eedc1-tnpnnmnlft.chromatic.com/?path=/story/all-components-tabs--with-actions)
[Production](https://storybook.polaris.shopify.com/?path=/story/all-components-tabs--with-actions&globals=polarisSummerEditions2023:true)

File: polaris-react/src/components/Tooltip/Tooltip.tsx [hunk 6]
```diff
@@ -86,10 +85,7 @@ export function Tooltip({
   onOpen,
   onClose,
 }: TooltipProps) {
-  const {polarisSummerEditions2023} = useFeatures();
-
-  const borderRadius =
-    borderRadiusProp || (polarisSummerEditions2023 ? '2' : '1');
+  const borderRadius = borderRadiusProp || '2';
```
Comment (sophschneider) [inline polaris-react/src/components/Tooltip/Tooltip.tsx:88]: [This PR](https://5d559397bae39100201eedc1-tnpnnmnlft.chromatic.com/?path=/story/all-components-tooltip--border-radius)
[Production](https://storybook.polaris.shopify.com/?path=/story/all-components-tooltip--border-radius&globals=polarisSummerEditions2023:true)

**Review by laurkim — APPROVED** [review]
🌟

**Pull request merged**